source/docs/advanced/payments/adyen.md

```diff
@@ -5,46 +5,67 @@ title: Adyen integration
 
 This page contains information about the different ways you can accept payments with [Adyen](https://www.adyen.com/) in your Front-Commerce application.
 
-## Magento2 module
+## Headless integration (no magento modules needed)
```

Comment:
Magento (with a capital M).

Also I'm unsure _Headless integration_ is the right term, I mean even with the
Magento2 module we are headless. Maybe:

```suggestion
## As a Front-Commerce payment (no Magento modules needed)
```

also before this title, an introduction to explain the main difference between the 2 options would be interesting.